chain/vm/runtime.go

```diff
@@ -342,7 +342,7 @@ func (rt *Runtime) Send(to address.Address, method abi.MethodNum, m vmr.CBORMars
 	if m != nil {
 		buf := new(bytes.Buffer)
 		if err := m.MarshalCBOR(buf); err != nil {
-			rt.Abortf(exitcode.SysErrInvalidParameters, "failed to marshal input parameters: %s", err)
+			rt.Abortf(exitcode.ErrSerialization, "failed to marshal input parameters: %s", err)
```

Comment:
This was a somewhat legitimate use, but I've changed this to be consistent with our policy elsewhere that parameter/result and state de/serialization is logically part of actor code, and so uses the actor `ErrSerialization`.